godel_msgs/srv/LoadSaveMotionPlan.srv

```diff
@@ -0,0 +1,13 @@
+# This service commands the blending service to load or save a motion plan from the file system
+int32 MODE_LOAD=0
+int32 MODE_SAVE=1
+
+int32 mode
```

Comment:
As this *service commands the blending service to load or save a motion plan*, would this perhaps be better named *command* or *action*?